<?xml version="1.0" encoding="utf-8"?>

<launch>
  <!-- Console launch prefix -->
  <arg name="launch_prefix" default=""/>
  <arg name="image" default="/astra/rgb/image_raw" />

  <!-- Config and weights folder. -->
  <arg name="openpose_weights_path"          default="$(find robot_vision_openvino)/models/openpose/"/>
  <arg name="openpose_weights_name"           default="human-pose-estimation.xml"/>

  <!-- Start darknet and ros wrapper -->
  <node pkg="robot_vision_openvino" type="openpose_ros" name="openpose_ros" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="weights_path"          value="$(arg openpose_weights_path)" />
    <param name="config_path"           value="$(arg openpose_weights_name)" />
    <remap from="camera/rgb/image_raw"  to="$(arg image)" />
  </node>

 <!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>
